tests/visual/config.js

```diff
@@ -0,0 +1,39 @@
+/* global module */
+
+'use strict';
+
+/**
+ * Set config options for BackstopJS
+ *
+ * @param viewports array Viewports to be tested.
+ * @param scenarios array Scenarios to be tested.
+ */
+const config = ( viewports, scenarios ) => {
+
+	return {
+		id: 'tenup-scaffold',
+		viewports: viewports,
+		scenarios: scenarios,
+		onBeforeScript: 'puppet/onBefore.js',
+		onReadyScript: 'puppet/onReady.js',
```

Comment:
Getting an warning here:
WARNING: script not found: /Users/timwright/Sites/tenup-scaffold-theme-test/wordpress/wp-content/themes/theme-scaffold/tests/visual/backstop_data/engine_scripts/puppet/onReady.js

Reply:
Added test:visual-init command to readme. Has to be ran first to initialize those files.